File: idade.js
let idade = 26

if (idade > 0 && idade < 12 )
    console.log( `Criança ${idade}`)

else if (idade > 13 && idade < 17)
    console.log( ` Adolescente ${idade}`)

else if  (idade > 17 && idade < 59)
    console.log( `adulto ${idade}`)

else if (idade >60)
    console.log(`Idoso ${idade}`)

else
    console.log( "Digito errado")


console.log(idade)

File: idade.rb
idade = gets.chomp.to_i

if idade > 0 && idade < 12 
    puts "Criança #{idade}"
elsif idade > 13 && idade < 17
    puts " Adolescente #{idade}"
elsif idade > 17 && idade < 59
    puts "Adulto #{idade}"
elsif idade > 60
    puts "Idoso #{idade}"

else
    puts "Digito errado"

end

